NGitLab.Tests/JobTests.cs

```diff
@@ -13,29 +13,32 @@ public class JobTests
 {
     internal static void AddGitLabCiFile(IGitLabClient client, Project project, int jobCount = 1, bool manualAction = false, string branch = null, bool pipelineSucceeds = true)
     {
-        var content = @"
-variables:
-  CI_DEBUG_TRACE: ""true""
-";
+        var content = """
+            variables:
+              CI_DEBUG_TRACE: "true"
+            
```

Comment:
```suggestion
```

Reply:
We need that "empty" line. Without it, subsequent string concatenations will result in invalid YAML.

Refer to [Microsoft's documentation about raw strings](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string):

`The newline before the closing quotes isn't included in the literal string.`